docs/index.md

```diff
@@ -1,21 +1,36 @@
 
-As such, the charm makes it easy for those looking to take control of their own agents whilst keeping operations simple, and gives them the freedom to deploy on the platform of their choice.
+## In this documentation
 
-For DevOps or SRE teams this charm will make operating Jenkins agents simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing changes, and supports scaling out for enterprise deployments.
+| | |
+|--|--|
+|  [Tutorials](https://charmhub.io/jenkins-agent/docs/tutorial-getting-started)</br>  Get started - a hands-on introduction to using the charm for new users </br> |  [How-to guides](https://charmhub.io/jenkins-agent/docs/how-to-configure-agent-node-label) </br> Step-by-step guides covering key operations and common tasks |
+| [Reference](https://charmhub.io/jenkins-agent/docs/reference-actions) </br> Technical information - specifications, APIs, architecture | [Explanation](https://charmhub.io/jenkins-agent/docs/explanation-workload) </br> Concepts - discussion and clarification of key topics  |
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as 
+the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. 
+Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) 
+to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to 
+either directly change something you see that's wrong, ask a question or make a suggestion about a potential change via 
+the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please 
+[file a bug](https://github.com/canonical/jenkins-agent-operator/issues).
 
 ## Project and community
 
-The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+The jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
 
 - [Code of conduct](https://ubuntu.com/community/code-of-conduct)
 - [Get support](https://discourse.charmhub.io/)
 - [Join our online chat](https://app.element.io/#/room/#charmhub-charmdev:ubuntu.com)
-- [Contribute](Contribute)
+- [Contribute](https://github.com/canonical/jenkins-agent-operator/blob/3e451213530aba783b892d231ce0f783a22ec303/CONTRIBUTING.md)
```

Comment:
How will this be loaded in charmhub? Real time? It this the blob corresponding to the main branch?

Reply:
I grabbed the permalink for the main branch CONTRIBUTING.md file. That link will take the reader off of Charmhub and onto GitHub, but another possibility would be to point to a "How to contribute" page on Charmhub that points to the CONTRIBUTING.md file.